src/components/Panel/Panel.tsx

```diff
@@ -4,13 +4,18 @@ import styled from "styled-components";
 export type PanelPadding = "none" | "xs" | "sm" | "md" | "lg" | "xl";
 export type PanelColor = "default" | "muted" | "transparent";
 
+type Orientation = "horizontal" | "vertical";
```

Comment:
We might need to export this globally as we are using it in multiple places

Reply:
right. done